handler/deployment_config_callback.go

```diff
@@ -181,14 +186,172 @@ func (h *handler) ReceiveDeleteDeploymentConfig(s *discordgo.Session, i *discord
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to connect to server over ssh",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		return
+	}
+	defer sshClient.Close()
+
+	// Create a new SCP client
+	scpClient, err := scp.NewClientBySSH(sshClient)
+	if err != nil {
+		log.Println(err)
+		if err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+			Type: discordgo.InteractionResponseChannelMessageWithSource,
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to connect to server over ssh",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		return
+	}
+
+	// Connect to the remote server
+	if err := scpClient.Connect(); err != nil {
+		log.Println(err)
+		if err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+			Type: discordgo.InteractionResponseChannelMessageWithSource,
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to connect to server over ssh",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		return
+	}
+	defer scpClient.Close()
+
+	// Upload yaml file
+	yamlFilePath, err := template.GenerateYaml(deploymentConfig)
+	if err != nil {
+		log.Println(err)
+	}
+	file, err := os.Open(yamlFilePath)
+	if err != nil {
+		log.Println(err)
+	}
+	filename := deploymentConfig.Name + "_" + deploymentConfig.ID + ".yaml"
+	remoteFilePath := filepath.Join(serverDeploymentFileRoot, filename)
+	permission := "0655"
+	if err := scpClient.CopyFromFile(context.Background(), *file, remoteFilePath, permission); err != nil {
+		log.Println(err)
+		if err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+			Type: discordgo.InteractionResponseChannelMessageWithSource,
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to copy file to server",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		return
+	}
+	template.RemoveYaml(yamlFilePath)
+
+	// create ssh session
+	session, err := sshClient.NewSession()
+	defer session.Close()
+	if err != nil {
+		log.Println(err)
+		if err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+			Type: discordgo.InteractionResponseChannelMessageWithSource,
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to connect to server over ssh",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		return
+	}
+
+	// kubectl apply file
+	cmd := "kubectl apply " + serverDeploymentFileRoot
+	result, err := session.Output(cmd)
+	if err != nil {
+		log.Println(err)
+		if err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+			Type: discordgo.InteractionResponseChannelMessageWithSource,
+			Data: &discordgo.InteractionResponseData{
+				Content: "❌ Failed to apply deployment config",
+			},
+		}); err != nil {
+			log.Println(err)
+		}
+		template.RemoveYaml(yamlFilePath)
+		return
+	}
+	// TODO: retrieve and send kubectl log to client
+	log.Println("[Kubectl Log]:\n", string(result))
```

Comment:
This one shouldn't be log into backend server... or our log server will be flooded...
Btw, logging is a REALLY EXPENSIVE SERVICE.

Reply:
yeah, it should be removed
 just for testing purposes